config/routes.rb

```diff
@@ -2,4 +2,9 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :places
   resources :varieties
+  resources :orders, only: [:index, :create]
+  get '/orders/new/:id', to: 'orders#new_order', as: "new_order"
+  post '/orders/create', to: 'orders#create', as: "create_order"
+  get '/orders/show_order', as: "show_order"
+  post '/orders/finish', to: 'orders#finish', as: 'finish_order'
 end
```

Comment:
Lo comenté en otro PR, pero si este es el que será mergeado, se mantiene el issue: Hace falta definir la ruta raíz o "home", porque sigue saliendo el "You're on rails":
![image](https://user-images.githubusercontent.com/530965/54372339-850c9380-4651-11e9-90c9-3a58284a9cbf.png)

Reply:
listo, la pagina de inicio es orders